travis_configure.sh

```diff
@@ -32,9 +32,11 @@ before_install()
     if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
         # jfrog dependency
         if [[ -n "$TRAVIS_TAG" ]]; then
-            curl -XGET -L -k 'https://api.bintray.com/content/jfrog/jfrog-cli-go/$latest/jfrog-cli-linux-amd64/jfrog?bt_package=jfrog-cli-linux-amd64' > /tmp/jfrog ;
-            chmod a+x /tmp/jfrog ;
-            sudo cp /tmp/jfrog /usr/local/bin/jfrog ;
+            if [[ ! -d $HOME/bin || ! -e $HOME/bin/jfrog ]]; then
```

Comment:
This is a bit redundant, `[[ -e $HOME/bin/jfrog ]]` should be enough, right? Also, why do you even bother to check? Who do you expect to install `jfrog` in the home directory for you?

Reply:
> This is a bit redundant, 

Sure, was thinking it would print the error like doing `ls` on a non-existing nested directory

Reply:
Yeah, but it's not important.